pkg/parser/sysl.wbnf

```diff
@@ -97,7 +97,7 @@ NativeDataTypes -> "int32" | "int64" | "int" |
 reference ->  \s* pkg=(NAME:"::" ".")? APPNAME;
 
 COMMENT_NO_NL -> "#" [^\n]*;
-COMMENT -> COMMENT_NO_NL "\n";
+COMMENT -> COMMENT_NO_NL \n+;
```

Comment:
Why is this change here? It is not described in the PR or the commit message, which is all about `http_method_comment`. Is this change meant to be here? (it looks like it was possibly a first attempt to fix the problem).

Reply:
Right, forgot to remove this. I'll delete the change along with resolving conflict.

Reply:
change added